internal/command/command.go

```diff
@@ -424,14 +431,40 @@ func UsageMetrics(command *cobra.Command, wg *sync.WaitGroup) {
 
 // GlobalFlags contains all global flags definitions.
 type GlobalFlags struct {
-	Filter           string
-	Format           string
-	Save             string
-	Host             string
-	HostNetworkKey   string
-	Log              string
-	Network          string
-	Yes              bool
-	ConfigPaths      []string
-	SkipVersionCheck bool
+	Filter                     string
+	Format                     string
+	Save                       string
+	Host                       string
+	HostNetworkKey             string
+	Log                        string
+	Network                    string
+	Yes                        bool
+	ConfigPaths                []string
+	SkipVersionCheck           bool
+	SkipContractMigrationCheck bool
+}
+
+func checkContractMigrations(state *flowkit.State, logger output.Logger, flow flowkit.Services) {
+	contractStatuses, err := validator.NewValidator(github.NewClient(nil).Repositories, flow.Network(), state, logger).GetContractStatuses()
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to validate contracts: %s", err))
+		return
+	}
+
+	for _, contract := range contractStatuses {
+		if contract.IsFailure() {
+			logger.Error(fmt.Sprintf(
+				"Contract %s has failed the last emulated migration\n"+
+					" - Account: %s\n"+
+					" - Contract: %s\n"+
+					" - Error: %s\n",
+				contract.ContractName,
+				contract.AccountAddress,
+				contract.ContractName,
+				contract.Error,
+			))
```

Comment:
Also - imo we should keep this message minimal, e.g. Just list all of the contracts that failed, if any, and link to the migration report.  It takes up to 1 week (and many CLI commands) to resolve these errors, so we should try not to pollute the user's terminal too much as this could become annoying.